src/StatusMsg.chpl

```diff
@@ -0,0 +1,24 @@
+module StatusMsg {
+    use Reflection;
+    use ServerConfig;
+    use Logging;    
+    use Message;
+    use MemoryMgmt;
+    use MultiTypeSymbolTable;
+    use MultiTypeSymEntry;
+    
+    private config const logLevel = ServerConfig.logLevel;
+    private config const logChannel = ServerConfig.logChannel;
+    const sLogger = new Logger(logLevel, logChannel);
+    
+    /**
+     * 
+     */
```

Comment:
I assume this is left over and can be deleted

Reply:
@jaketrookman ah, good catch, wanted to add some docs there, will do